packages/app/src/runner/index.ts

```diff
@@ -292,7 +292,7 @@ function setSpecForDriver (spec: SpecFile) {
  * a Spec IFrame to load the spec's source code, and
  * initialize Cypress on the AUT.
  */
-function runSpecE2E (config, spec: SpecFile) {
+async function runSpecE2E (config, spec: SpecFile) {
```

Comment:
This was something interesting I noticed. `visitBlankPage` was a hanging promise. I believe these changes resolves this.